third_party/string_view.h

```diff
@@ -1,6 +1,2 @@
-#if __cplusplus >= 201703L || defined(__has_include) && __has_include(<string_view>)
```

Comment:
https://github.com/cquery-project/cquery/issues/390

On Mac OS X, Xcode somehow includes `<string_view>` so these lines cannot be removed

Reply:
I wanted to sync my repository before submitting and this issue popped up, so this was just a quickfix.
This change should be a separate issue, which I assume others will encounter soon.